examples/experimental/dagster-airlift/dagster_airlift/airflow_utils.py

```diff
@@ -304,3 +307,34 @@ def get_leaf_specs(specs: List[AssetSpec]) -> List[AssetSpec]:
         if key not in downstreams_per_key
     ]
     return leaf_specs
+
+
+def toposort_flatten(specs: List[AssetSpec]) -> List[AssetSpec]:
```

Comment:
there's an existent toposort_flatten in the codebase. That's what I meant.

Reply:
ah